be/src/formats/parquet/schema.cpp

```diff
@@ -46,49 +46,92 @@ std::string ParquetField::debug_string() const {
     return ss.str();
 }
 
-static bool is_group(const tparquet::SchemaElement& schema) {
-    return schema.num_children > 0;
-}
+//static bool is_group(const tparquet::SchemaElement& schema) {
+//    return schema.num_children > 0;
+//}
 
-static bool is_list(const tparquet::SchemaElement& schema) {
-    return schema.__isset.converted_type && schema.converted_type == tparquet::ConvertedType::LIST;
+static bool is_group(const tparquet::SchemaElement* schema) {
```

Comment:
what's the point of changing reference to pointer?

Reply:
In order to return `StatusOr<SchemaElement*>`